docs/insomnia/subscription-management.md

```diff
@@ -5,6 +5,18 @@ category: "Insomnia Subscriptions"
 category-url: subscription-management
 ---
 
+## View Subscription Details
+
+Users can view their current plan, the total number of seats in the subscription, and the number of seats currently in use by navigating to the—depending on your plan—"Account Settings" or "Enterprise Controls" page and then to the "Billing" section.
+
+![Account Settings, Enterprise Controls, and Billing sections in the Insomnia web GUI](../assets/images/billing-enterprise-controls-menu.png)
+
+The **Current seat usage** displays the combined total of users across all Organizations and pending invites. Invites remain valid in perpetuity, and need to be rescinded in order to reclaim the seat.
```

Comment:
@marckong I'd like to add a bit more here explaining how invites can be deleted to free up seats. Is there a way for users to do that today _without_ navigating to individual orgs?

Reply:
if the user is on enterprise plan, they can go to the license page